redbot/cogs/mod/settings.py

```diff
@@ -85,35 +79,155 @@ async def hierarchy(self, ctx: commands.Context):
+
+        if mention_spam["kick"]:
+            if max_mentions >= mention_spam["kick"]:
+                mismatch_message += _("\nAutowarn is equal or higher than autokick.")
+
+        if mention_spam["ban"]:
+            if max_mentions >= mention_spam["ban"]:
+                mismatch_message += _("\nAutowarn is equal to or higher than autoban.")
+
+        await self.config.guild(ctx.guild).mention_spam.warn.set(max_mentions)
+        await ctx.send(
+            _(
+                "Autowarn for mention spam enabled. "
+                "Anyone mentioning {max_mentions} or more different people "
+                "in a single message will be autowarned.\n{mismatch_message}"
+            ).format(max_mentions=max_mentions, mismatch_message=mismatch_message)
+        )
+
+    @mentionspam.command()
     @commands.guild_only()
-    async def banmentionspam(self, ctx: commands.Context, max_mentions: int = 0):
+    async def kick(self, ctx: commands.Context, max_mentions: int = 0):
+        """
+        Sets the autokick conditions for mention spam.
+
+        Users will be kicked if they send any messages with contain more than
+        `<max_mentions>` mentions.
+
+        `<max_mentions>` must be at least 3. Set to 0 to disable.
+        """
+        mention_spam = await self.config.guild(ctx.guild).mention_spam.all()
+        if not max_mentions:
+            if not mention_spam["kick"]:
+                return await ctx.send(_("Autokick for mention spam is already disabled."))
+            await self.config.guild(ctx.guild).mention_spam.kick.set(False)
+            return await ctx.send(_("Autokick for mention spam disabled."))
+
+        if max_mentions < 3:
+            return await ctx.send(_("`<max_mentions>` must be 3 or higher to autokick."))
+
+        mismatch_message = ""
+
+        if mention_spam["warn"]:
+            if max_mentions <= mention_spam["warn"]:
+                mismatch_message += _("\nAutokick is equal to or lower than autowarn.")
+
+        if mention_spam["ban"]:
+            if max_mentions >= mention_spam["ban"]:
+                mismatch_message += _("\nAutokick is equal to or higher than autoban.")
+
+        await self.config.guild(ctx.guild).mention_spam.kick.set(max_mentions)
+        await ctx.send(
+            _(
+                "Autokick for mention spam enabled. "
+                "Anyone mentioning {max_mentions} or more different people "
+                "in a single message will be autokicked.\n{mismatch_message}"
+            ).format(max_mentions=max_mentions, mismatch_message=mismatch_message)
+        )
+
+    @mentionspam.command()
+    @commands.guild_only()
+    async def ban(self, ctx: commands.Context, max_mentions: int = 0):
         """Set the autoban conditions for mention spam.
 
         Users will be banned if they send any message which contains more than
         `<max_mentions>` mentions.
 
         `<max_mentions>` must be at least 5. Set to 0 to disable.
```

Comment:
Again, this kinda just seems like an arbitrary number (even though it is existing code).

Reply:
Fixed, changed ban/kick/warn to at least be 1. So people don't attempt to action on...every message...